packages/ui/src/kit/VTable/VTableHeadCell/VTableHeadCell.vue

```diff
@@ -1,21 +1,97 @@
 <template>
   <th :class="vTableHeadCellCssClass">
-    <slot />
+    <a
+      href="#"
+      class="v-table-head-cell-sort-wrapper"
+      @click.prevent="onClick"
+    >
+      <slot />
+      <svg-atom
+        v-if="isSortable"
+        class="v-table-head-cell-sort-icon"
+        :name="sortIcon"
+        width="8"
+        height="8"
+      />
+    </a>
   </th>
 </template>
 
 <script>
 import ThemeMixin from '@/mixins/ThemeMixin';
+import SvgAtom from '@/atom/svg-atom/svg-atom';
+
+const SORTING_DIRECTIONS = {
+  DESC: 'desc',
+  ASC: 'asc',
+};
 
 export default {
   name: 'VTableHeadCell',
+
+  props: {
+    sortingBy: {
+      type: String,
+      default: null,
+    },
+
+    sortingValue: {
```

Comment:
and no need to pass `sortingValue` to.
head cell does not know about any values, only sort direction